pkg/util/secretutil/secret.go

```diff
@@ -28,6 +28,7 @@ import (
 	"sort"
 	"strings"
 
+	"golang.org/x/crypto/pkcs12"
```

Comment:
Move this after line 34 and group with `corev1 "k8s.io/api/core/v1"` as they're external packages